docs/python/python-tutorial.md

```diff
@@ -227,24 +227,24 @@ plt.show()                   # Display the plot
 
 Next, try running the file in the debugger using the "Python: Current file" configuration as described in the last section. (If you still have `"stopOnEntry": true` in that configuration, you need to select the run command again to continue.)
 
-Unless you're using an Anaconda distribution or have previously installed the matplotlib package, you should see the message, "ModuleNotFoundError: No module named 'matplotlib'". Such a message indicates that the required package isn't available in your system.
+Unless you're using an Anaconda distribution or have previously installed the `matplotlib` package, you should see the message, "ModuleNotFoundError: No module named 'matplotlib'". Such a message indicates that the required package isn't available in your system.
 
-To install the matplotlib package (which also installs numpy), stop the debugger and run **Python: Create Terminal** from the Command Palette, which opens a command prompt for your selected interpreter. Then enter the following commands as appropriate for your operating system (commands may require elevation if the Python interpreter is installed in a protected area of the file system):
+To install the `matplotlib` package (which also installs `numpy` as a dependency), stop the debugger and run **Python: Create Terminal** from the Command Palette. This will open a command prompt for your selected interpreter. Then enter the following commands as appropriate for your operating system (commands may require elevation if the Python interpreter is installed in a protected area of the file system):
 
 > **Note**: If you are unable to install the package, please [file an issue on GitHub](https://github.com/Microsoft/vscode-docs/issues) so we can help you investigate.
 
 ```bash
-# pip[3] install commands may require elevation
+# pip install commands may require elevation
 
 # macOS
-pip3 install matplotlib
+python -m pip install matplotlib
 
 # Windows
-pip install matplotlib
+py -m pip install matplotlib
 
-# Linux
+# Linux (Debian)
 sudo apt-get install python3-tk
-pip3 install matplotlib
+python -m pip install matplotlib
 ```
```

Comment:
Again, please confirm pip3 vs. pip on all Linux systems.

Reply:
I can confirm `python -m pip` is confirmed over bare `pip` (or `pip3`)

Reply:
As with the question for MacOS, would it be python3 -m pip instead of python -m pip? I'm personally unclear (not using those OSs myself) when using python3 or python applies.